src/sidebar/components/menu.js

```diff
@@ -179,7 +167,7 @@ export default function Menu({
             onClick={closeMenu}
             onKeyDown={handleMenuKeyDown}
           >
-            <MenuKeyboardNavigation visible={openedByKeyboard}>
+            <MenuKeyboardNavigation visible={true}>
```

Comment:
nb. Although `visible` is always `true` in this component, `MenuItem` still uses it and sets it depending on whether the submenu is visible or not. We still want that behavior.